src/server.js

```diff
@@ -36,13 +36,16 @@ const Translators = require('./translators');
 const SearchEndpoint = require('./searchEndpoint');
 const WebEndpoint = require('./webEndpoint');
 const ExportEndpoint = require('./exportEndpoint');
+const Recognizer = require('./recognizer');
 
 const app = module.exports = new Koa();
 app.use(cors);
 app.use(bodyParser({ enableTypes: ['text', 'json']}));
 app.use(_.post('/web', WebEndpoint.handle.bind(WebEndpoint)));
 app.use(_.post('/search', SearchEndpoint.handle.bind(SearchEndpoint)));
 app.use(_.post('/export', ExportEndpoint.handle.bind(ExportEndpoint)));
+app.use(_.post('/recognize/getUploadParams', Recognizer.handleUpload.bind(Recognizer)));
```

Comment:
Since this isn't actually creating state or doing any work, this could be a GET instead of a POST. And then just `GET /recognize/uploadParams`.